reflections.md

```diff
@@ -4,14 +4,16 @@ With any large project that you've undertaken, especially one which requires a l
 We're providing a set of suggested reflection prompts / questions that you can use to consider how things went with your capstone project, and we encourage you to add your own prompts as well!
 
 ## Reflection prompts
-1. What went well during the capstone project?
-1. What didn't go as well?
+1. What did you learn about:
+  * Researching new technologies and investigating their applicability to a specific project?
+  * Building a software system comprised of multiple interacting pieces?
+  * Prioritization when building a proof of concept or MVP?
+  * Debugging unfamiliar technologies and complex interactions between components?
+
 1. What aspects of the project were more difficult than anticipated?
 1. What aspects were less difficult than anticipated?
-1. What features in your project were not possible in the time frame provided? Were any of them critical to the success of the project?
-
-1. How did your project plan shift as the project progressed?
-1. If any project features took longer than anticipated to complete, how did you modify the rest of your planned features to accommodate having less time?
-1. How did your intended plan of action line up with the reality?
+1. What features in your project were not possible in the time frame provided? Why?
```

Comment:
I think you're right. It probably needs some rephrasing if it's going to be meaningful.

I was hoping to get students to consider which features took longer than they had planned for and thus left them without enough time to complete other (hopefully less critical) things.